router/comptue_routes.gno

```diff
@@ -180,7 +181,7 @@ func findCandidatePools() []PoolWithMeta {
 			token0Path,
 			token1Path,
 			pFee,
-			token0Path + ":" + token1Path,
+			ufmt.Sprintf("%s:%s", token0Path, token1Path),
```

Comment:
why not use string builder here?

Reply:
in this case, these is no significant difference between using strings.Builder and formatting function. so I use latter one.